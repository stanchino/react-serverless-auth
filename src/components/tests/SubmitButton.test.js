import { matchSnapshot } from "./shared-examples";

import { SubmitButton } from "..";

describe("SubmitButton", () => {
    it("matches the snapshot", () => matchSnapshot(SubmitButton, { form: { pristine: false, submitting: false } }));
});