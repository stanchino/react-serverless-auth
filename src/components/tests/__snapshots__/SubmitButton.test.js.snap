// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubmitButton matches the snapshot 1`] = `
<button
  disabled={false}
  type="submit"
/>
`;
